bin/desi_tile_redshifts

```diff
@@ -39,26 +39,27 @@ Not supported yet: multiple tiles on a single night in a single call:
 
 """
 
-#- Parse command line quickly for --help before slower imports
+# - Parse command line quickly for --help before slower imports
 import argparse
+
 p = argparse.ArgumentParser()
 p.add_argument("-n", "--night", type=int, nargs='+', help="YEARMMDD nights")
 p.add_argument("-t", "--tileid", type=int, help="Tile ID")
 p.add_argument("-e", "--expid", type=int, nargs='+', help="exposure IDs")
 p.add_argument("-g", "--group", type=str, required=True,
-        help="cumulative, pernight, perexp, or a custom name")
+               help="cumulative, pernight, perexp, or a custom name")
 p.add_argument("--explist", type=str,
-        help="file with columns TILE NIGHT EXPID to use")
+               help="file with columns TILE NIGHT EXPID to use")
 p.add_argument("--nosubmit", action="store_true",
-        help="generate scripts but don't submit batch jobs")
+               help="generate scripts but don't submit batch jobs")
 p.add_argument("--batch-queue", type=str, default='realtime',
-        help="batch queue name")
+               help="batch queue name")
 p.add_argument("--batch-reservation", type=str,
-        help="batch reservation name")
+               help="batch reservation name")
 p.add_argument("--batch-dependency", type=str,
-        help="job dependencies passed to sbatch --dependency")
+               help="job dependencies passed to sbatch --dependency")
 p.add_argument("--system-name", type=str,
-        help="batch system name, e.g. cori-haswell, cori-knl, perlmutter-gpu")
+               help="batch system name, e.g. cori-haswell, cori-knl, perlmutter-gpu")
```

Comment:
Minor, but for consistency, the standard for the other scripts is that this parser logic should move into `args = desispec.scripts.tile_redshifts.parse()`, and that there should be a `desispec.scripts.tile_redshifts.main(args)`.  That `main` could be itself a super-simple to call `generate_tile_redshift_scripts(**args.__dict__)`, since that function also gets used by the pipeline without going through argument parsing etc.

Reply:
Thanks, I have implemented these changes. I will note for thoroughness that this approach does come at a small speed cost in outputting "`desi_tile_redshifts --help`" because it now imports all the packages when importing `desispec.scripts.tile_redshifts`, but I really don't mind.